indexer/src/test/java/org/pdxfinder/commandline/FinderCommandLineTest.java

```diff
@@ -60,22 +63,44 @@ public void setUp() {
     }
 
     @Test public void load_givenTransform_When_exportAllisCalled_Then_callsTransformer() throws IOException {
-        String[] args = {"--data-dir=path/", "--all"};
+        String[] args = {"--data-dir=path/", "-c=mut"};
+        int exitCode = new CommandLine(transform).execute(args);
+        assertEquals(0, exitCode);
+        verify(this.finderTransformer).run(
+                any(File.class),
+                any(null),
```

Comment:
Replace all instances of `any(null)` with  `any()` - you shouldn't pass `null` as a parameter to methods